src/commands/synthetics/api.ts

```diff
@@ -220,7 +220,8 @@ const uploadMobileApplicationPart = (request: (args: AxiosRequestConfig) => Axio
       request
     )
 
-    const quotedEtag = resp.headers.etag as string
+    // Azure part-upload does not return ETag -- set to empty string
```

Comment:
I still think it's useful to say _why_ is an empty string OK in our very specific case
```suggestion
    // Azure part-upload does not return ETag headers, and our backend ignores it for Azure
```